src/vaadin-context-menu.html

```diff
@@ -295,6 +296,7 @@
 
             /**
              * Event name to listen for opening the context menu.
+             * @type {string}
              */
             openOn: {
               type: String,
```

Comment:
Should this accept nullish values?
Nullish values are handled simply by removing existing listener (if any), and we have the public `open(Event)` method as an alternative way for the developer to control the opening.

Reply:
Same as above, I would like to use `string` or better `keyof HTMLElementEventMap`.

However there is no functionality in `gen-typescript-declarations` to automatically declare arbitrary custom events. So for now it should be ok to use `string` I guess. 

Generally, I'm trying to consistently use `@type {string}` when initial value is string.